docs/source/models/concept_models/vivarium_mncnh_portfolio/maternal_disorders_module/module_document.rst

```diff
@@ -98,7 +100,9 @@ Incidence, mortality, YLDs, and YLLs due to cause-specific maternal disorders.
 3.0 Assumptions and limitations
 ++++++++++++++++++++++++++++++++
 
-* There are many other maternal disorders which we do not plan to individually model. 
+* We assume that all deaths due to maternal disorders occur during the intrapartum period prior to the postpartum period although especially for causes of deaths such as "late maternal deaths," this may not be the case.
+* We do not model morbidity due to the subcauses included in the :ref:`Residual maternal disorders <2021_cause_residual_maternal_disorders_mncnh>` cause
```

Comment:
I don't understand this one; should we say the thing about applying to everyone rather than identifying cases instead?